### PR TITLE
feat(WorkflowStatus): u#2032 t#3721 create status endpoint

### PR DESCRIPTION
**Pull request by tdelatorre**
### How to test
- [x] Review code
- [x] Tests are green

Import postman files
- [ ] Create workflow status and check everything is correct
- [ ] You may try to create a status in the front using the branch ` us/2032/task/3724/create-status`

### CODE REVIEW

File: python/apps/taiga/src/taiga/workflows/repositories.py [hunk 1]
```diff
@@ -165,51 +166,99 @@ def _apply_filters_to_workflow_status_queryset(
...
+
 ##########################################################
 # WorkflowStatus - create workflow status
 ##########################################################
 
 
 def create_workflow_status_sync(
     name: str,
-    slug: str,
+    slug: str | None,
```
Comment (yamila-moreno) [inline python/apps/taiga/src/taiga/workflows/repositories.py:204]: in which scenarios the slug is provided?

File: python/apps/taiga/src/taiga/workflows/repositories.py [hunk 2]
```diff
@@ -165,51 +166,99 @@ def _apply_filters_to_workflow_status_queryset(
...
     )
+    if slug:
+        status.slug = slug
+
+    status.save()
+
+    return status
 
 
 create_workflow_status = sync_to_async(create_workflow_status_sync)
```
Comment (yamila-moreno) [inline python/apps/taiga/src/taiga/workflows/repositories.py:223]: is it still necessary?

File: python/apps/taiga/src/taiga/workflows/repositories.py (hunk 1 above)
Comment (tdelatorre) [reply python/apps/taiga/src/taiga/workflows/repositories.py:204]: when template is upload

File: python/apps/taiga/src/taiga/workflows/repositories.py (hunk 2 above)
Comment (tdelatorre) [reply python/apps/taiga/src/taiga/workflows/repositories.py:223]: yes. Because we create statuses with an inital template

**Review by bameda — APPROVED** [review]

**Pull request merged**